modules/core/core-api/src/main/java/com/enonic/xp/project/ProjectRoles.java

```diff
@@ -0,0 +1,18 @@
+package com.enonic.xp.project;
```

Comment:
I don't see the need for it to be part of core-api.
Can it be part of core-content ?